src/lib/components/NavbarMobile.svelte

```diff
@@ -59,23 +59,23 @@
 		<div class="navbar">
 			<ul>
 				<li class:active={activeSection == 'home'}>
-					<a href="#" on:click={scrollToTop}>Home</a>
+					<a href="#/" on:click={scrollToTop}>Home</a>
```

Comment:
Just checking, is there supposed to be an extra slash here?

Reply:
the terminal was throwing errors when I had just the '#' and changing it to '#/" fixed them? I can change it back if you want?